server/api/products.js

```diff
@@ -57,11 +57,9 @@ router.get('/:productId', (req, res, next) => {
     where: {
       id: req.params.productId,
     },
-    include: [
-      {
-        model: Review
-      }
-    ]
+    include: {
```

Comment:
actually, it was correct the first way - `includes` is an array containing an object.